app/views/organizations/_organization_filters.html.erb

```diff
@@ -38,7 +38,7 @@
           <div class="SelectMenu-list">
             <% view_options.each do |name| %>
               <label class="SelectMenu-item" role="menuitemradio" aria-checked="<%= active_view_option == name %>" tabindex="0">
-                <%= radio_button_tag "view", name, active_view_option == name, "hidden": "true", "data-autosubmit": "true" %>
+                <%= radio_button_tag "view", name, active_view_option == name, "hidden": "true" %>
```

Comment:
Does this still trigger when clicking in Safari? (I think `change` catches this here: https://github.com/education/classroom/pull/2427/files#diff-121f9615472808b2e0a69f8ea7a51c25R12)

Reply:
Yep! Still works chrome and safari. And now it works in Firefox too!

I have no idea about IE and Edge....but I don't have a free way (that I know of) to check them.